quick-start/e2e/specs/create/create.ts

```diff
@@ -168,7 +170,28 @@ export default function() {
       browser.wait(EC.presenceOf(entityPage.toast));
       browser.wait(EC.stalenessOf(entityPage.toast));
     });
-
+    
+    it ('should toggle pii button for product entity', function() {
+      console.log('ad');
+      browser.executeScript('window.document.getElementsByClassName("edit-start")[1].click()');
+      browser.wait(EC.visibilityOf(entityPage.entityEditor));
+      expect(entityPage.entityEditor.isPresent()).toBe(true);
+      let skuProperty = entityPage.getPropertyByPosition(1);
+      expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(true);
```

Comment:
why there are multiple verification with conflicting values? One is true and one is false? Shouldn't it only be true?